assignment.js

```diff
@@ -113,6 +136,47 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+const playerAChoice = "scissors"
+const playerBChoice = "paper"
+
+function whoWins(playerA, playerB) {
```

Comment:
@justinbudlow nice use of implementing this in a function (even though we hadn't yet fully covered functions at this point)